source/agora/utils/InetUtils.d

```diff
@@ -322,10 +322,6 @@ version (Windows)
 
         assert(extractHostAndPort("seed.bosagora.io/blabla")
             == HostPortTup("seed.bosagora.io", 80, HostType.Domain));
-
-        // Negative tests
-        assert(extractHostAndPort("blab/blabla")
-            == HostPortTup("", 0, HostType.Invalid));
```

Comment:
Why is this removed ?

Reply:
In the newest version of the implementation "blab/blabla" is valid: the implementation will try to detect if the url has any schema or not, and if not then "http" is assumed. 

I can add more negative tests though, if you like.

Reply:
Well then this part of the diff should be squashed in the previous commit. Because the previous commit introduce a test, and this commit removes that diff. Since this commit does not touch the function that is tested, either the previous commit does not pass tests, or the test shouldn't be removed. From what you're telling me it sounds like the former ?

Reply:
sorry my bad, fixed all the issues by now